BackendApartmentReservation/BackendApartmentReservation/Apartments/ApartmentsController.cs

```diff
@@ -25,5 +31,15 @@ public async Task<IEnumerable<string>> Get()
             _logger.LogInformation("Initial step");
             return await Task.FromResult(new[] { "flat1", "flat2" });
         }
+
+        [HttpGet]
+        [Route("availableApartmentsRooms")]

```

Comment:
make route `trips/{tripId}/apartment/rooms/available`

Also, you should check rooms only for a single apartment, which can be gotten from trip.
Trip -> Destination Office -> Apartment

Reply:
This could be a separate method (eg `GetTripApartments`)